# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libcss/gcc33-hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' libcss-0.1.0/src/css_cat.c libcss-0.1.0-gcc-3.3.1/src/css_cat.c
--- libcss-0.1.0/src/css_cat.c	2001-02-13 22:59:27.000000000 +0100
+++ libcss-0.1.0-gcc-3.3.1/src/css_cat.c	2003-08-19 22:43:44.000000000 +0200
@@ -60,14 +60,14 @@
 
 #define STCODE(p,a,b,c,d) ((p)[0] == a && (p)[1] == b && (p)[2] == c && (p)[3] == d)
 
-u_long long offset = 0x0;
-u_long long offset_end = 0x249a4;
+unsigned long long offset = 0x0;
+unsigned long long offset_end = 0x249a4;
 
 /**
  *
  */
 
-static void super_seek (int fd, u_long long offset)
+static void super_seek (int fd, unsigned long long offset)
 {
 	int i;
 
